isis/src/base/objs/SpiceRotation/SpiceRotation.cpp

```diff
@@ -402,6 +404,48 @@ namespace Isis {
   }
 
 
+  void SpiceRotation::LoadCache(json &isdRot){
+    if (p_source != Spice) {
+        throw IException(IException::Programmer, "SpiceRotation::LoadCache(json) only support Spice source", _FILEINFO_);
+    }  
+
+    p_timeFrames.clear();
+    p_TC.clear();
+    p_cache.clear();
+    p_cacheTime.clear();
+    p_cacheAv.clear();
+    p_hasAngularVelocity = false;
+    m_frameType = PCK; 
+    
+    // Load the full cache time information from the label if available
+    p_fullCacheStartTime = isdRot["CkTableStartTime"].get<double>();
+    p_fullCacheEndTime = isdRot["CkTableEndTime"].get<double>();
+    p_fullCacheSize = isdRot["CkTableOriginalSize"].get<double>(); 
+    p_cacheTime = isdRot["EphemerisTimes"].get<std::vector<double>>(); 
+    p_timeFrames = isdRot["TimeDependentFrames"].get<std::vector<int>>();
+
+    for (auto it = isdRot["Quaternions"].begin(); it != isdRot["Quaternions"].end(); it++) {
+        std::vector<double> quat = {it->at(0).get<double>(), it->at(1).get<double>(), it->at(2).get<double>(), it->at(3).get<double>()};
+        p_cache.push_back(quat);
+    }
+
+    
+    bool hasConstantFrames = isdRot.find("ConstantFrames") != isdRot.end();
+    if (hasConstantFrames) {
+      p_constantFrames = isdRot["ConstantFrames"].get<std::vector<int>>();
+      p_TC.resize(9);
+      ident_c((SpiceDouble( *)[3]) &p_TC[0]); 
```

Comment:
We don't seem to have a way to acquire the Frame Chain info SpiceRotation wants from Ale at the moment so this is a tempory solution, make it an identity matrix.

Reply:
Adams changes to ALE (https://github.com/USGS-Astrogeology/ale/pull/274) took care of this issue